### PR TITLE
fix: minor style changes

### PR DESCRIPTION
**Pull request by hyunjiariel**

### CODE REVIEW

File: src/components/BoardColumn/ColumnHeader.vue [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <template>
-  <div class="flex justify-between pb-4">
+  <div
+    class="bg-gradient-to-r from-teal-400 to-blue-500 focus:from-pink-500 focus:to-orange-500 text-white font-semibold px-6 py-3 rounded-tr-lg rounded-tl-lg flex justify-between pb-4"
+  >
```
Comment (Alejocg) [inline src/components/BoardColumn/ColumnHeader.vue:4]: Can we use here the new color palette? for buttons we´re using variations of indigo-800 or rose-800 and for nav-bar/footer, gray-500

File: src/components/BoardColumn/TaskCard.vue [hunk 2]
```diff
@@ -86,7 +87,7 @@ const formRef = ref(null);
 const rules = {
   task: {
     required: true,
-    validator(rule, value) {
+    validator(value) {
```
Comment (evagapi) [inline src/components/BoardColumn/TaskCard.vue]: This change breaks the validation logic. Is there any reason behind it? Even if we aren't using the rule, the value is passed as a second parameter so we shouldn't remove the `rule` parameter.

Comment (evagapi) [reply src/components/BoardColumn/TaskCard.vue]: Can we revert all the changes in this file?

File: tailwind.config.cjs [hunk 3]
```diff
@@ -2,7 +2,8 @@
 module.exports = {
   content: ["./index.html", "./src/**/*.{vue,js}"],
   theme: {
-    extend: {},
+    extend: {
```
Comment (evagapi) [inline tailwind.config.cjs]: Can we revert the changes in this file? Thanks!

File: src/components/AddColumnForm.vue [hunk 4]
```diff
@@ -3,8 +3,8 @@
     class="cursor-pointer w-80 rounded-lg bg-white flex justify-center items-center gap-2 flex-col border-8 border-slate-300 hover:bg-slate-100"
     @click="showModal = true"
   >
-    <div class="text-3xl font-bold text-center">
-      <p class="text-6xl">+</p>
+    <div class="text-3xl font-bold text-center text-gray-500">
+      <p class="text-6xl text-gray-500">+</p>
```
Comment (evagapi) [inline src/components/AddColumnForm.vue]: We don't need this if it is already defined in the parent, do we?
```suggestion
      <p class="text-6xl">+</p>
```

File: src/components/TodoList.vue [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="main-full-height pt-8">
+  <div class="main-full-height pt-7">
```
Comment (evagapi) [inline src/components/TodoList.vue:2]: Good catch @hyunjiariel! 🏅 🙌

File: src/components/BoardColumn/ColumnHeader.vue [hunk 6]
```diff
@@ -1,5 +1,7 @@
 <template>
-  <div class="flex justify-between pb-4">
+  <div
+    class="bg-gradient-to-r from-indigo-500 via- to-indigo-800 text-white font-semibold px-6 py-3 rounded-tr-lg rounded-tl-lg flex justify-between pb-4"
```
Comment (evagapi) [inline src/components/BoardColumn/ColumnHeader.vue]: What is the purpose of the class `via-`?
`py-3` isn't working properly right?

**Review by evagapi — CHANGES_REQUESTED** [review]
Looks nicer, but we have introduced some bugs (i left some comments)🐞

File: src/components/BoardColumn/TaskCard.vue (hunk 2 above)
Comment (hyunjiariel) [reply src/components/BoardColumn/TaskCard.vue]: oh actually I did not mean to change that.. how can I revert it?

File: tailwind.config.cjs (hunk 3 above)
Comment (hyunjiariel) [reply tailwind.config.cjs]: sure!

File: src/components/BoardColumn/ColumnHeader.vue (hunk 6 above)
Comment (hyunjiariel) [reply src/components/BoardColumn/ColumnHeader.vue]: I actually took 'py-3' from the tailwind 'gradient'document, will try to remove it if it does not make a difference :)

File: src/components/AddColumnForm.vue (hunk 4 above)
Comment (hyunjiariel) [reply src/components/AddColumnForm.vue]: true! thanks for finding it :)

File: src/components/BoardColumn/ColumnHeader.vue (hunk 6 above)
Comment (hyunjiariel) [reply src/components/BoardColumn/ColumnHeader.vue]: oh and I forgot to delete 'via-'

Comment (hyunjiariel) [reply src/components/BoardColumn/ColumnHeader.vue]: I just checked and if I deleted py-3 it does not look nice as there is no space so left it!

File: src/components/BoardColumn/TaskCard.vue [hunk 7]
```diff
@@ -107,6 +108,7 @@ function handleSubmit() {
       message.success("Task has been successfully edited");
     })
     .catch(() => {
+      // eslint-disable-next-line max-lines
```
Comment (evagapi) [inline src/components/BoardColumn/TaskCard.vue]:
```suggestion
```

File: src/components/BoardColumn/TaskCard.vue [hunk 8]
```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable max-lines -->
```
Comment (evagapi) [inline src/components/BoardColumn/TaskCard.vue]:
```suggestion
```

**Review by evagapi — APPROVED** [review]
LGTM 👍
